image.yaml

```diff
@@ -0,0 +1,133 @@
+    - name: "io.openshift.s2i.scripts-url"
+      value: "image:///usr/local/s2i"
+    - name: "maintainer"
+      value: "Red Hat"
+envs:
+    - name: HTTPS_ENABLE_HTTP2
+      value: "true"
+    - name: SCRIPT_DEBUG
+      description: If set to true, ensures that the bash scripts are executed with the -x option, printing the commands and their arguments as they are executed.
+      example: "true"
+ports:
+    - value: 8443
+modules:
+      repositories:
+          - name: cct_module
+            git:
+                  url: https://github.com/jboss-openshift/cct_module.git
+                  ref: 0.44.0
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/luck3y/jboss-eap-modules.git
+                  ref: CLOUD-3918
+          - name: jboss-eap-7-image
+            git:
+                  url: https://github.com/luck3y/jboss-eap-7-image.git
+                  ref: CLOUD-3918
+          - name: wildfly-cekit-modules
+            git:
+                  url: https://github.com/luck3y/wildfly-cekit-modules.git
+                  ref: CLOUD-3918
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "11"
+          - name: jboss.container.maven
+            version: "8.2.3.6"
+          - name: eap-74-beta-env-latest
+          - name: jboss.container.eap.setup
+          - name: eap-install-cleanup
+          # This one indirectly installs common logging by creating a link to $BIN_HOME/bin/launch
+          # so must be after jboss.container.eap.cd
+          - name: jboss.container.maven.default
+          - name: dynamic-resources
+          - name: jboss.container.eap.s2i.galleon
+          - name: jboss.container.eap.galleon
+          - name: jboss.container.eap.galleon.config.ee
+          - name: jboss.container.eap.galleon.build-settings
+            version: "public"
+          - name: jboss.container.eap.openshift.modules
+          - name: os-eap-activemq-rar
+            version: "1.1"
+          - name: jboss.container.eap.amq6
+          - name: jboss.container.java.jvm.bash
+          - name: jboss.container.eap.launch
+          - name: os-eap70-sso
+          - name: jboss.container.wildfly.launch.admin
+          - name: jboss.container.wildfly.launch.access-log-valve
+          - name: jboss.container.wildfly.launch-config.config
+          - name: jboss.container.wildfly.launch-config.os
+          - name: jboss.container.wildfly.launch.datasources
+          - name: jboss.container.wildfly.launch.extensions
+          - name: jboss.container.wildfly.launch.json-logging
+          - name: jboss.container.wildfly.launch.jgroups
+          - name: jboss.container.wildfly.launch.filters
+          - name: jboss.container.wildfly.launch.logger-category
+          - name: jboss.container.wildfly.launch.mp-config
+          - name: jboss.container.wildfly.launch.os.node-name
+          - name: jboss.container.wildfly.launch.tracing
```

Comment:
EAP 7.4.0.Beta no longer provides MicroProfile OpenTracing. This module should be removed aiui